sdk/python/kfp/components/_python_op.py

```diff
@@ -304,6 +310,16 @@ def annotation_to_type_struct(annotation):
             output_spec._passing_style = None
             output_spec._return_tuple_field_name = field_name
             outputs.append(output_spec)
+    elif isinstance(return_ann, dict): # Deprecated dict-based way of declaring multiple outputs. Was only used by the @acomponent decorator
+        import warnings
+        warnings.warn("The ability to specify multiple outputs using the dict syntax has been deprecated and will be removed soon after release 0.1.32. Please use typing.NamedTuple to declare multiple outputs.")
```

Comment:
Is that possible to break this into multiple lines so that they can fit in the screen?

Reply:
Done